install.sh

```diff
@@ -721,8 +721,9 @@ pushd .
     CXX=${compiler} ${cmake_executable} ${cmake_common_options[@]} ${cmake_client_options[@]} -DCPACK_SET_DESTDIR=OFF -DCMAKE_INSTALL_PREFIX="${rocm_path}" \
     -DCMAKE_PREFIX_PATH="${rocm_path};${rocm_path}/hip;$(pwd)/../deps/deps-install;${cuda_path};${cmake_prefix_path}" \
     -DCMAKE_SHARED_LINKER_FLAGS="" \
-    -DCMAKE_EXE_LINKER_FLAGS=" -Wl,--disable-new-dtags -Wl,--rpath,${rocm_path}/lib:${rocm_path}/lib64" \
+    -DCMAKE_EXE_LINKER_FLAGS="" \
     -DROCM_DISABLE_LDCONFIG=ON \
+    -DCMAKE_SKIP_RPATH=TRUE \
```

Comment:
Don't set a variable to ="" just leave it out entirely, we don't hide what might get set earlier in the code 
-DCMAKE_SHARED_LINKER_FLAGS and -DCMAKE_EXE_LINKER_FLAGS lines should just be removed if you  don't want them set to a vaue.

Reply:
Thanks for the comments, since we are removing entire runpath, rpath configurations, as per inputs this was intentionally made as empty in the proposed changes in this version.
 
But Yes will take your comment and try to understand if it is already having any values and we are removing by this action. Will get back on this

Reply:
One can set unrelated shared linker flags that are still required, just like DCMAKE_EXE_LINKER_FLAGS thus code that clears variables is not expected or good practice